tests/local/test_json.py

```diff
@@ -38,3 +42,52 @@ def test_lookup_foo_bar(self):
     def test_lookup_foo_baz(self):
         result = self.cache.lookup('local:directory:foo/unknown')
         self.assertEqual([], result)
+
+
+class JsonLibraryTest(unittest.TestCase):
+
+    config = {
+        'local': {
+            'media_dir': path_to_data_dir(''),
+            'data_dir': path_to_data_dir(''),
+            'playlists_dir': b'',
+            'library': 'json',
+        },
+    }
+
+    def setUp(self):  # noqa: N802
+        self.library = json.JsonLibrary(self.config)
+
+    def _create_tracks(self, count):
+        for i in xrange(count):
```

Comment:
Use `range()` instead of `xrange()` to be a bit more ready for porting to Python 3. We've already replaced `xrange()` everywhere else in the code base.